### PR TITLE
Fix typos and grammar issues in comments

### PR DESCRIPTION
**Pull request by alexandear**
This PR fixes issues in comments:

* Typos in words.
* Use a comma before 'so' if it connects two independent clauses (unless they are closely connected and short). 
* This word is normally spelled with a hyphen.
* Use a comma before 'or' if it connects two independent clauses (unless they are closely connected and short). 
* Use an instead of 'a' if the following word starts with a vowel sound, e.g. 'an article', 'an hour'. 
* Did you mean past? 
* Consider using either the past participle filed or the present participle filing here. 
* Consider using many. 
* When '64-bit' is used as a modifier, it is usually spelled with a hyphen. 
* The past participle is required after "must be", alternatively you could omit the "be".

### CODE REVIEW

File: params.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"strconv"
 )
 
-// Params represents a set of parameters that gets passed to a request.
+// Params represents a set of parameters that gets past to a request.
```
Comment (Syfaro) [inline params.go:10]: I think this was correct as-is (passed, not past).

**Review by Syfaro — CHANGES_REQUESTED** [review]
Mostly looks good, I think there's one thing that wasn't quite correct.

I also quite substantially updated the master branch, so it might take a little bit of messing around to get it into a state where it can be merged again, sorry!

File: params.go (hunk 1 above)
Comment (alexandear) [reply params.go:10]: agree, will revert

**Comment by alexandear** [comment]
@Syfaro I updated PR, please recheck

**Review by Syfaro — APPROVED** [review]
Looks good, thanks!

**Pull request merged**